### PR TITLE
feat: add trackings per device

### LINKED ISSUE #1: Send activity per device

### PR DESCRIPTION
**Pull request by konstanto**
Extract devices and submit trackings per device

Fixes #1

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -5,107 +5,102 @@
...
+    devices = get_devices(cur)
+
+    for index, device in enumerate(devices):
+      events = get_events_for_device(device[0], cur)
+      print(f"{index + 1} / {len(devices)} Sending {len(events)} events to ActivityWatch for device {device[0]} - {device[1]}")
+      if len(events) > 0:
+        send_to_activitywatch(events, device)
+        
+def get_devices(database_connection):
+  query = """
```
Comment (ErikBjare) [inline main.py]: Two-space indentation? Don't see that much in Python-land :P

File: main.py [hunk 2]
```diff
@@ -5,107 +5,102 @@
...
+    SELECT
+      DISTINCT(ZSOURCE.ZDEVICEID) as deviceId,
+	  ZSYNCPEER.ZMODEL as deviceModel
```
Comment (ErikBjare) [inline main.py:30]: Mixed tabs and spaces

File: main.py [hunk 3]
```diff
@@ -5,107 +5,102 @@
...
-
-
 def main() -> None:
     dbfile = _get_db_path()
     print(f"Reading from database file at {dbfile}")
-
-    # Open the database
     conn = sqlite3.connect(_get_db_path())
-
-    # Set journal mode to WAL.
```
Comment (ErikBjare) [inline main.py:53]: Why remove these comments?

File: main.py [hunk 4]
```diff
@@ -5,107 +5,102 @@
...
     bucket = f"aw-watcher-android_aw-import-screentime_{hostname}"
 
     aw = ActivityWatchClient(client_name="aw-import-screentime")
     aw.client_hostname = hostname
-
-    # buckets = aw.get_buckets()
-    # if bucket in buckets.keys():
-    #     ans = input("Bucket already found, overwrite? (y/N): ")
-    #     if ans == "y":
-    #         aw.delete_bucket(bucket, force=True)
```
Comment (ErikBjare) [inline main.py:93]: This should probably remain, since there is no incremental import (yet).

**Review by ErikBjare — COMMENTED** [review]
Should be formatted with `black`, but otherwise looks good!

One question though: Does this remove support for macOS devices? (are they returned by `get_devices`?)

**Comment by abegong** [comment]
Would it be helpful to pick up this PR? I'd also like to be able to split and filter events by device.

**Comment by ErikBjare** [comment]
@abegong Yes, merging this. Thanks for the ping!

**Pull request merged**